src/Microsoft.SymbolStore.Client.Tests/SymbolLocatorTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using FileFormats;
+using FileFormats.PDB;
+using FileFormats.PE;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.SymbolStore.Client
+{
+    public partial class Tests
+    {
+        [Fact]
+        public async void SymbolServerEndToEnd()
+        {
+            SymbolLocator locator = new SymbolLocator(new SymbolCache(CacheLocation), new ISymbolServer[] { CreateWindowsSymbolServer() });
+
+            Task<string>[] tasks = new Task<string>[2];
```

Comment:
How about something like this?

Task peTask = (async () => ValidatePEFile(locator, await locator.FindPEFileAsync(PEFilename, PEFileTimestamp, PEFileSize).Result))();
Task pdbTask = (async () => { ValidatePdb(locator, await locator.FindPdbAsync(PDBFileName, PDBGuid, PDBAge).Result))();
await Task.WhenAll(peTask, pdbTask);